release notes/v0.54.0.md

```diff
@@ -0,0 +1,304 @@
+
+let file;
+let csvRecords;
+(async function () {
+    file = await open('data.csv');
+
+    // The `csv.parse` function consumes the entire file at once, and returns
+    // the parsed records as a SharedArray object.
+    csvRecords = await csv.parse(file, {delimiter: ','})
+})();
+
+
+export default async function() {
+    // The csvRecords a SharedArray. Each element is a record from the CSV file, represented as an array
+    // where each element is a field from the CSV record.
+    //
+    // Thus, `csvRecords[scenario.iterationInTest]` will give us the record for the current iteration.
+    console.log(csvRecords[scenario.iterationInTest])
+}
+```
+
+</details>
+
+<details>
+<summary> Expand to see an example of streaming a CSV file line-by-line.</summary>
+
+```javascript
+import { open } from 'k6/experimental/fs'
+import csv from 'k6/experimental/csv'
+
+export const options = {
+    iterations: 10,
+}
+
+let file;
+let parser;
+(async function () {
+    file = await open('data.csv');
+    parser = new csv.Parser(file);
+})();
+
+export default async function() {
+    // The parser `next` method attempts to read the next row from the CSV file.
+    //
+    // It returns an iterator-like object with a `done` property that indicates whether
+    // there are more rows to read, and a `value` property that contains the row fields
+    // as an array.
+    const {done, value} = await parser.next();
+    if (done) {
+        throw new Error("No more rows to read");
+    }
+
+    // We expect the `value` property to be an array of strings, where each string is a field
+    // from the CSV record.
+    console.log(done, value);
+}
+```
+
+</details>
+
+### New `k6 cloud upload` command for uploading test files to the cloud [`#3906`](https://github.com/grafana/k6/pull/3906)
+
+Although we continue to refine and improve the cloud service to handle better the uploading of test files,
+as well as the overall user experience, we've added a new `k6 cloud upload` command that replaces the
+`k6 cloud --upload-only` flag used so far, which is now considered deprecated.
+
+### gRPC module updates driven by contributors
+
+#### New `discardResponseMessage` option
+
+[#3877](https://github.com/grafana/k6/pull/3877) and [#3820](https://github.com/grafana/k6/pull/3820) add a new option for gRPC module `discardResponseMessage` which allows users to discard the messages received from the server.
+
+```javascript
+const resp = client.invoke('main.RouteGuide/GetFeature', req, {discardResponseMessage: true});
+```
+
+This reduces the amount of memory required, and the amount of garbage collection, which reduces the load on the testing machine and can help produce more reliable test results.
+
+Thank you, @lzakharov!
+
+#### New argument `meta` for gRPC's stream callbacks
+
+[#3801](https://github.com/grafana/k6/pull/3801) adds a new argument `meta` to gRPC's stream callback, which handles the timestamp of the original event (e.g. when message has been received).
```

Comment:
```suggestion
[#3801](https://github.com/grafana/k6/pull/3801) adds a new argument `meta` to gRPC's stream callback, which handles the timestamp of the original event (for example, when a message has been received).
```

We try to avoid using e.g. and i.e. based on our writing style guide.